### DISCUSSION TITLE
Port Scanner - Enable / allow selecting multiple port profiles

### DISCUSSION THREAD

**Discussion by Dr4571c-S0lu710n5** [BornToBeRoot/NETworkManager#1976] [Ideas]
In the Port Scanner enable the possibility to select multiple port profiles. Par example: holding Ctrl + selecting profiles.

**Comment by BornToBeRoot** [comment]
You can also add your own profile in the settings:
![image](https://user-images.githubusercontent.com/16019165/222860942-9f536526-a405-4cb8-9a12-8d9cd8ae08cf.png)

**Comment by BornToBeRoot** [comment]
Done via #1979

![PortScanner_PortProfiles](https://user-images.githubusercontent.com/16019165/222865650-94e93f4e-e7a4-4e07-8d93-75c2fc923b48.gif)
